test/functional/wallet_assumeutxo.py

```diff
@@ -111,7 +115,11 @@ def run_test(self):
 
         # Mine more blocks on top of the snapshot that n1 hasn't yet seen. This
         # will allow us to test n1's sync-to-tip on top of a snapshot.
-        self.generate(n0, nblocks=100, sync_fun=self.no_op)
+        for i in range(100):
+            if i % 3 == 0:
+                self.mini_wallet.send_to(from_node=n0, scriptPubKey=address_to_scriptpubkey(w_address), amount=1 * COIN)
+                self.mini_wallet.send_to(from_node=n0, scriptPubKey=address_to_scriptpubkey(w2_address), amount=10 * COIN)
```

Comment:
nit: could call `address_to_scriptpubkey()` only once outside the for loop.

Reply:
done